esphome/components/tx20/sensor.py

```diff
@@ -0,0 +1,38 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome import pins
+from esphome.components import sensor
+from esphome.const import CONF_ID, CONF_WIND_SPEED, CONF_PIN, \
+    CONF_WIND_DIRECTION_DEGREES, UNIT_KILOMETER_PER_HOUR, \
+    UNIT_EMPTY, ICON_WEATHER_WINDY, ICON_SIGN_DIRECTION
+
+tx20_ns = cg.esphome_ns.namespace('tx20')
+Tx20Component = tx20_ns.class_('Tx20Component', cg.Component)
+
+CONFIG_SCHEMA = cv.Schema({
+    cv.GenerateID(): cv.declare_id(Tx20Component),
+    cv.Optional(CONF_WIND_SPEED):
+        sensor.sensor_schema(UNIT_KILOMETER_PER_HOUR, ICON_WEATHER_WINDY, 1).extend(),
```

Comment:
```suggestion
        sensor.sensor_schema(UNIT_KILOMETER_PER_HOUR, ICON_WEATHER_WINDY, 1),
```

extend can be removed because nothing is added.

Reply:
Right - both of them removed.